Detectors/TRD/base/src/FeeParam.cxx

```diff
@@ -195,7 +195,7 @@ void FeeParam::Copy(FeeParam& p) const
 }
 
 //_____________________________________________________________________________
-int FeeParam::getPadRowFromMCM(int irob, int imcm) const
+int FeeParam::getPadRowFromMCM(int irob, int imcm)
```

Comment:
Minor style question: why did you remove the const?

Reply:
I seem to remember it was a hack because I got frustrated with something in the digit class. Maybe let me go back and fix that properly.